src/app/views/NoteCreate.svelte

```diff
@@ -217,6 +293,14 @@
           disabled={$loading || signaturePending}>
           {#if signaturePending}
             <i class="fa fa-circle-notch fa-spin" />
+          {:else if options?.publish_at && options?.publish_at > now()}
+            {#if new Date(options.publish_at * 1000).toDateString() === new Date().toDateString()}
+              Scheduled for {new Date(options.publish_at * 1000).toLocaleTimeString()}
+            {:else}
+              Scheduled for {new Date(options.publish_at * 1000).toLocaleDateString()} at {new Date(
+                options.publish_at * 1000,
+              ).toLocaleTimeString()}
+            {/if}
```

Comment:
Instead of lots of all caps and numbers, let's just change this to "Schedule"

Reply:
maybe add a small text below the button for when it is actually scheduled, just the date in local date format, ie 12-23-2025 at 20:00

Reply:
Let's leave it off for now, until people ask for it. They just set the date, so the confirmation shouldn't be necessary